### PR TITLE
Moving code editor component from inside of codeblock to make it easier to style

### PR DESCRIPTION
**Pull request by tahmidefaz**
Screenshot:

image

### CODE REVIEW

File: src/components/APIDoc/CodeSamples.tsx [hunk 1]
```diff
@@ -59,39 +59,37 @@ export const CodeSamples: React.FunctionComponent<CodeSampleProps> = ({parameter
...
+                </ClipboardCopyButton>
+            </Flex>
+        </Flex>
+        <CodeEditor
+            isDarkTheme={true}
+            isLineNumbersVisible={true}
+            isReadOnly={true}
+            // isMinimapVisible={false}
+            // isLanguageLabelVisible={true}
+            // isCopyEnabled={true}
```
Comment (josejulio) [inline src/components/APIDoc/CodeSamples.tsx:89]: Could you remove unused properties?

**Comment by epwinchell** [comment]
Looks great

File: src/components/APIDoc/CodeSamples.tsx (hunk 1 above)
Comment (tahmidefaz) [reply src/components/APIDoc/CodeSamples.tsx]: Removed! Good catch!

File: src/components/APIDoc/CodeSamples.tsx [hunk 2]
```diff
@@ -59,39 +59,37 @@ export const CodeSamples: React.FunctionComponent<CodeSampleProps> = ({parameter
...
+            code={code.toString()}
+            language={Language.javascript}
```
[the remark this answers is not in the document]
Comment (tahmidefaz) [reply src/components/APIDoc/CodeSamples.tsx]: Added proper language highlighting to each snippet now. :+1:

**Review by josejulio — APPROVED** [review]

**Pull request merged**